Sources/SwiftSDKGenerator/SwiftSDKRecipes/LinuxRecipe.swift

```diff
@@ -105,13 +105,20 @@ public struct LinuxRecipe: SwiftSDKRecipe {
   }
 
   public func applyPlatformOptions(toolset: inout Toolset, targetTriple: Triple) {
-    toolset.swiftCompiler = Toolset.ToolProperties(extraCLIOptions: [
-      "-use-ld=lld",
-      "-Xlinker",
-      "-R/usr/lib/swift/linux/",
-    ])
+    var swiftCompilerOptions = ["-Xlinker", "-R/usr/lib/swift/linux/"]
```

Comment:
I think `-Xlinker -R/usr/lib/swift/linux/` may not be necessary any more, but want to do more testing before removing it.